modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -7,10 +7,11 @@
 
 As a network administrator, you can configure a network attachment definition of type `cnv-bridge` to provide Layer-2 networking to pods and virtual machines.
 
-[NOTE]
-====
-The network attachment definition must be in the same namespace as the pod or virtual machine.
-====
+.Prerequisites
+
+* The network attachment definition must be in the same namespace as the pod or virtual machine.
+
+* Optional: The node must support nftables and the `nft` binary must be deployed to enable MAC spoof check.
```

Comment:
I think for this to work , nftables are mandatory to have. Optional looks confusing.

Reply:
Removed the word "Optional"

Reply:
Well, if no one uses mac-spoof-check, it is indeed optional. The sentence says that if one wants to use it, it is needed.
But I do not really mind if this gets removed.